change/@graphitation-supermassive-591f1b18-d5b9-4887-9c4d-701fbdf5e085.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "[SUPERMASSIVE] export AST types",
```

```suggestion
  "comment": "Export AST types",
```

don't put [SUPERMASSIVE] to change comments - it will already be recorded to the package.